Information as an Asset Through Business Intelligence and Data Analytics (#89)
ippontech/blog-usa: posts/information-as-an-asset-through-business-intelligence-and-data-analytics.md

---
authors:
- Rita Barlow
tags:
- Data Analytics
- Business Intelligence
- Data 
- Big Data 
- Information 
date: 2019-02-05T12:21:50.000Z
title: "Information as an Asset Through Business Intelligence and Data Analytics"

---
## Make Better Decisions Faster

> Business Intelligence and Data Analytics are critical practices to ensure your organization is running efficiently and proactively. In order to determine _how_ these fundamentals of data practices can help improve operations within your organization, we must first be able to determine the difference between the two, and answer how they can be utilized moving forward.

Reviewer: ```suggestion
Business Intelligence and Data Analytics are critical practices to ensure your organization is running efficiently and proactively. To determine _how_ these fundamentals of data practices can help improve operations within your organization, we must first be able to determine the difference between the two and answer how they can be utilized moving forward.
```

Author: Roger that

**Business Intelligence (BI)** refers to the technologies, applications, and practices used to collect, integrate, analyze, and visualize your organization's data as valuable information.  This technology-driven process is used to help executives, managers, and other data consumers in your organization make informed business decisions. BI allows your organization to identify business opportunities, gauge productivity, and optimize spending for areas like sales and marketing by predicting customer behavior and analyzing trends over time.

> **Data Analytics (DA)** refers to the act of converting unstructured or raw data into a comprehensive or informational format that can be used to support your organization at all levels. Your organization will be able to make quality decisions, improve planning and forecasting, enable consistent data across your enterprise, improve speed, accessibility, ease of use, and ultimately lower costs and increase operational efficiency. The benefits to how Data Analytics can improve your organization are nearly endless. The following are a couple examples where Data Analytics can be applied to your data to make it a powerful informational asset.

Reviewer: ```suggestion
**Data Analytics (DA)** refers to the act of converting unstructured or raw data into a comprehensive or informational format that can be used to support your organization at all levels. Your organization will be able to make quality decisions, improve planning and forecasting, enable consistent data across your enterprise, improve speed, accessibility, ease of use, and ultimately lower costs and increase operational efficiency. The benefits of how Data Analytics can improve your organization are nearly endless. The following are a couple of examples where Data Analytics can be applied to your data to make it a powerful informational asset.
```

Author: Corrected.

> - **Customer Experience:** Data Analytics allows you to get to know your customers by compiling and understanding data regarding their habits, preferences, and needs. This enables your organization to tailor offerings and solutions that will optimize your customers' experience and outcomes, making them more likely to return to your organization in the future. This could be applied to product offerings, services, geographical locations of stores, frequency of contact, medium of contact, etc.

Reviewer: ```suggestion
- **Customer Experience:** Data Analytics allows you to get to know your customers by compiling and understanding data regarding their habits, preferences, and needs. This enables your organization to tailor offerings and solutions that will optimize your customers' experience and outcomes, making them more likely to return to your organization in the future. Data analytics could be applied to product offerings, services, geographical locations of stores, frequency of contact, and medium of contact for example.
```

Author: Corrected.

![](https://github.com/ritabarlow/blog-usa/blob/master/images/2019/01/BI_VS_Data_Analytics.png) 

Both of these practices use data as information to help you make decisions with slight subtleties. Through the use of BI, leaders in your organization can leverage existing data and present it in a way that allows your organization to learn from past mistakes and build on past successes. Replicating what works and changing what has not. As a subset of Business Intelligence, Data Analytics allows your organization to be truly intelligent by anticipating developments, driving innovation, and accurately making predictions that will help your organization in the future. To fully utilize data as an asset in your organization you need both Business Intelligence and Data Analytics working together.

## I want to invest in data analytics, but how do I get started? 

> Now that you’ve realized that you have all of this powerful information at your organization’s fingertips you are wondering what to do next.  As in anything else, the first step in solving any problem is identifying the issue, or to be frank, admitting that your organization even has an issue. Sure you have data, you likely generate massive amounts of it daily for your customers, about your customers, about products, expenses, etc. So what are the first steps in turning that data into valuable information for your organization? How do you approach your boss, or your boss’ boss and inform them that you’d like a huge chunk of their budget to invest in data analytics? How do you convince them that an investment in foundational data practices will provide sustainable growth or an enterprise view of the organization and give them a competitive advantage over others in their industry? It’s a tough conversation to approach, especially if a significant amount of data cleanup is involved to start really benefiting from a data analytics platform. After all, no one likes it when you call their baby ugly. Your angle … how much are you already spending by NOT investing in your data? Can you afford to keep manually manipulating data month after month, export after export, spreadsheet after spreadsheet? How much longer can you afford making poor business decisions based on month old data? Making a case for an investment in data analytics is as easy as making the case to NOT invest in data analytics. 

Reviewer: ```suggestion
Now that you’ve realized that you have all of this powerful information at your organization’s fingertips you are wondering what to do next.  As in anything else, the first step in solving any problem is identifying the issue, or to be frank, admitting that your organization even has an issue. Sure you have data, you likely generate massive amounts of it daily for your customers, about your customers, about products, and expenses. So what are the first steps in turning that data into valuable information for your organization? How do you approach your boss, or your boss’ boss and inform them that you’d like a huge chunk of their budget to invest in data analytics? How do you convince them that an investment in foundational data practices will provide sustainable growth or an enterprise view of the organization and give them a competitive advantage over others in their industry? It’s a tough conversation to approach, especially if a significant amount of data cleanup is involved to start really benefiting from a data analytics platform. After all, no one likes it when you call their baby ugly. Your angle: how much are you already spending by NOT investing in your data? Can you afford to keep manually manipulating data month after month, export after export, spreadsheet after spreadsheet? How much longer can you afford to make poor business decisions based on month old data? Making a case for an investment in data analytics is as easy as making the case to NOT invest in data analytics. 
```

Author: Corrected.

> The internet is inundated with resources for how savvy analytics from your data once you get organizational buy-in.  There is no shortage of big data technologies, analytics and BI platforms and opinions out there to take your money cruising down the data analytics highway.  In fact, it’s almost too easy to drop tons of cash on a shiney new platform and cross your fingers that when you run that report each month you are getting trusted and reliable information.  Getting the right answers from your data is almost like weight loss, there is no quick-win if you want it done properly.  You need to start with foundational data practices, clean up data at the source, assess fit for use within your organization, and put some core governance practices around your data to ensure that your investment stays reliable and accessible. It’s important to realize that what may work for your CIO friend at Company A may not work for you at your organization. The data needs for each organization vary vastly on the needs of that organization and how they intend to use their data moving forward.  I would strongly encourage your “start” to be an assessment of your organization’s informational needs and current state.  Determine where you are and where you need to be before jumping head first into a technology suite or shiney platform that is “guaranteed to solve your data needs”.  Every organization does not need a Masareti get to the finish line, sometimes a good ol’ Toyota Camry will do exactly what you need. 

Reviewer: ```suggestion
The internet is inundated with resources for how savvy analytics from your data once you get organizational buy-in.  There is no shortage of big data technologies, analytics and BI platforms and opinions out there to take your money cruising down the data analytics highway.  It’s almost too easy to drop tons of cash on a shiny new platform and cross your fingers that when you run that report each month, you are getting trusted and reliable information.  Getting the right answers from your data is almost like weight loss, there is no quick-win if you want it done correctly.  You need to start with foundational data practices, clean up data at the source, assess fit for use within your organization, and put some core governance practices around your data to ensure that your investment stays reliable and accessible. It’s important to realize that what may work for your CIO friend at Company A may not work for you at your organization. The data needs for each organization vary vastly on the needs of that organization and how they intend to use their data moving forward.  I would strongly encourage your “start” to be an assessment of your organization’s informational needs and current state.  Determine where you are and where you need to be before jumping head first into a technology suite or shiny platform that is “guaranteed to solve your data needs”.  Every organization does not need a Maserati to get to the finish line; sometimes a Toyota Camry will do exactly what you need. 
```

Author: Corrected.

> For more information on data governance and the importance of data quality in your organization please refer to the following post by Justin Risch, [Decoupling Data Governance](https://blog.ippon.tech/decoupling-data-governance/ "Decoupling Data Governance").

Reviewer 2: ```suggestion
For more information on data governance and the importance of data quality in your organization please refer to the following post by Justin Risch, [Decoupling Data Governance](https://blog.ippon.tech/decoupling-data-governance/).
```

Reviewer 2: This is marked "resolved" but I don't see a commit to fix the link.
